docs/topics/secured-mission-nodejs-adapter.adoc

```diff
@@ -0,0 +1,50 @@
+= SSO Adapter Configuration
+
+The SSO adapter is the _client side_, or client to the SSO server, component that enforces security on the web resources. In this specific case, it is the Greeting service.
+
+
+.Enacting Security Example {NodeJS} Code
+[source,javascript,options="nowrap",subs="attributes+"]
+----
+const express = require(‘express’);
```

Comment:
You don't mention this line in your description of the code below, but I think it's probably important. The `keycloak-connect` module acts as [connect **middleware**](https://github.com/senchalabs/connect), which integrates nicely with express. You might even consider mentioning this in the introductory text.